ui/packages/tidb-dashboard-for-dbaas/src/apps/Monitoring/metricsQueries.ts

```diff
@@ -47,7 +48,7 @@ function transformColorByExecTimeOverview(legendLabel: string) {
   }
 }
 
-const monitoringItems = [
+const monitoringItems: MetricsQueryType[] = [
```

Comment:
Specifying the `MetricsQueryType` instead of inference can help the ts compiler knows the type was correct. Otherwise, the `string` won't fit the `ColorType`.